backend/src/routes/sessions.js

```diff
@@ -1,6 +1,17 @@
 import express from 'express';
+import Restaurant from '../mongo/models/Restaurant';
 const router = express.Router();
 
 // TODO: Handlers...
+// GET:
+router.get('/testCard', async (req, res) => {
+  const testCard = new Restaurant({
+    name: 'Lonestar',
+    location: 'Owen G Glen Building',
+    cuisine: 'European',
+    price: '5',
```

Comment:
I'm not too familiar with Mongo schema types but did you define the type for `cuisine` to be an array of strings in `backend/src/mongo/models/Restaurant.js`? If so, should it then be `cuisine: ['European']`? The same goes for `price` too?

Reply:
Good call - I'll check that out. I didn't see any specific definition of types in the backend, but I made the Restaurant schema for this PR so maybe I should add some strings in the schema definition

Reply:
@jemimahmartinez I checked the restaurant schema and the response from the server - it looks like when using new Restaurant({...}) it automatically converts fields into the correct format.